recipes/jags/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
Good practice: add `set -e` at the top here, so that if the `configure` or `make` command fails, it'll give up rather than trying to continue.